_events/sr2023/virtual-tech-day-december.md

```diff
@@ -0,0 +1,29 @@
+---
+title: December Virtual Tech Day
+date: 2022-12-03 10:00:00 Europe/London
+end_date: 2022-12-03 17:00:00 Europe/London
+layout: event
+type: tech-day
+location: Discord
+---
+
+Tech Days are opportunities for teams to spend a whole day working on their
+robot with lots of help available. They’re also an opportunity to see how other
+teams are doing or get more direct help with their robots.
+
+The aim of this day is to help teams out by having guaranteed presence from us
+blueshirts. We'll be around to answer questions about the competition, give kit
+support, assist with the microgames, and assist with strategy ideas.
+
+Our SR2023 December Tech Day is being hosted on our Discord server.
```

Comment:
do we want to link to the discord docs page?

Reply:
d1d7d24